pkgs/servers/search/elasticsearch/plugins.nix

```diff
@@ -94,10 +94,10 @@ in {
   elasticsearch_kopf = esPlugin rec {
     name = "elasticsearch-kopf-${version}";
     pluginName = "elasticsearch-kopf";
-    version = "1.5.7";
+    version = "2.1.1";
```

Comment:
Ignore my last comment. Got confused for a moment. 
We should backport this to 17.03